include/maliput/utility/generate_dot.h

```diff
@@ -45,5 +46,18 @@ namespace utility {
 /// @throws common::assertion_error When @p os is nullptr.
 void GenerateDotStream(const routing::graph::Graph& graph, std::ostream* os);
 
+/// Generates a string representation in @p os of the @p graph and @p route using the DOT language.
+///
+/// routing::graph::Edges are created as bidirectional, and labels are added to indicate api::Segment::id().
+/// routing::graph::Nodes only indicate their routing::graph::NodeId. The @p route is indicated by changing the color
+/// of edges where it lays.
+///
+/// @param graph The routing::graph::Graph to serialize.
+/// @param route The routing::Route to highlight on top of the @p graph.
+/// @param os A pointer to a std::ostream to serialize the @p graph. It must not be nullptr.
+/// @throws common::assertion_error When @p os is nullptr.
```

Comment:
Does it also throw if @p route is not part of @p graph?

Reply:
[19ad84e](https://github.com/maliput/maliput/pull/639/commits/19ad84ed8ff47c49bb87d90dc4c53217485dadea)